cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -513,48 +525,59 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             actualMinutesStr = '00'.substring(0, 2 - actualMinutesStr.length) + actualMinutesStr;
             return hoursStr + ':' + actualMinutesStr;
         },
-        convertTimeLimitToMinutes: function(time_limit) {
-            var time = time_limit.split(':');
-            var total_time = (parseInt(time[0]) * 60) + parseInt(time[1]);
-            return total_time;
+        convertTimeLimitToMinutes: function(timeLimit) {
+            var time = timeLimit.split(':');
+            var totalTime = (parseInt(time[0], 10) * 60) + parseInt(time[1], 10);
+            return totalTime;
         },
         getRequestData: function() {
-            var is_time_limited;
-            var is_practice_exam;
-            var is_proctored_exam;
-            var time_limit = this.getExamTimeLimit();
-            var exam_review_rules = this.$('.field-exam-review-rules textarea').val();
+            var isTimeLimited;
+            var isPracticeExam;
+            var isProctoredExam;
+            var isOnboardingExam;
+            var timeLimit = this.getExamTimeLimit();
+            var examReviewRules = this.$('.field-exam-review-rules textarea').val();
 
             if (this.$('input.no_special_exam').is(':checked')) {
-                is_time_limited = false;
-                is_practice_exam = false;
-                is_proctored_exam = false;
+                isTimeLimited = false;
+                isPracticeExam = false;
+                isProctoredExam = false;
+                isOnboardingExam = false;
             } else if (this.$('input.timed_exam').is(':checked')) {
-                is_time_limited = true;
-                is_practice_exam = false;
-                is_proctored_exam = false;
+                isTimeLimited = true;
+                isPracticeExam = false;
+                isProctoredExam = false;
+                isOnboardingExam = false;
             } else if (this.$('input.proctored_exam').is(':checked')) {
-                is_time_limited = true;
-                is_practice_exam = false;
-                is_proctored_exam = true;
+                isTimeLimited = true;
+                isPracticeExam = false;
+                isProctoredExam = true;
+                isOnboardingExam = false;
             } else if (this.$('input.practice_exam').is(':checked')) {
-                is_time_limited = true;
-                is_practice_exam = true;
-                is_proctored_exam = true;
+                isTimeLimited = true;
+                isPracticeExam = true;
+                isProctoredExam = true;
+                isOnboardingExam = false;
+            } else if (this.$('input.onboarding_exam').is(':checked')) {
+                isTimeLimited = true;
+                isPracticeExam = true;
```

Comment:
is an onboarding exam a practice exam?

Stretch nit: can we refactor these sets as individual logical expressions? This is way verbose for some really simple logic

Reply:
No, it's not. Thanks for catching that.

I'll work on a refactor.